website/docs/validation/volumes.md

```diff
@@ -182,6 +196,49 @@ Usage
 kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper-library/master/library/pod-security-policy/volumes/samples/psp-volume-types/example_allowed.yaml
 ```
 
+</details>
+<details>
```

Comment:
NOTE ONLY: Opened https://github.com/open-policy-agent/gatekeeper-library/issues/400 as we might want to exclude `AdmissionReview` from being used as documentation samples.